great_expectations/render/types/__init__.py

```diff
@@ -299,6 +299,9 @@ def __str__(self):
         )
         return string
 
+    def __eq__(self, other):
+        return str(self) == str(other)
+
```

Comment:
this was necessary because the tests in `tests/render/test_page_renderer.py` were being done between two `RenderedStringTemplateContent` objects, which were not able to be compared directly in some cases.  This `__eq__` method makes the comparison explicit.